scss/_root.scss

```diff
@@ -167,7 +167,7 @@
   --#{$prefix}table-striped-hover-bg-factor: #{$table-striped-hover-bg-factor};
 
   // Breadcrumb-specific styles
-  --#{$prefix}breadcrumb-divider-filter: none;
+  --#{$prefix}breadcrumb-divider-filter: #{$breadcrumb-divider-filter};
 
   // Navigation-specific styles
   --#{$prefix}nav-link-active-color: #{$body-color};
```

Comment:
This one has been forgotten. I'd say we should have `$nav-link-active-color` too.